kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/FileNames.java

```diff
@@ -44,9 +45,6 @@ public enum DeltaLogFileType {
   private static final Pattern COMPACTION_FILE_PATTERN =
       Pattern.compile("\\d+\\.\\d+\\.compacted\\.json");
 
-  /** Example: 00000000000000000001.dc0f9f58-a1a0-46fd-971a-bd8b2e9dbb81.json */
-  private static final Pattern UUID_DELTA_FILE_REGEX = Pattern.compile("(\\d+)\\.([^\\.]+)\\.json");
```

Comment:
We should add this later, and correctly, when we support catalogManaged tables that have staged uuid delta files.